cmake/modules/RootMacros.cmake

```diff
@@ -553,8 +564,15 @@ function(ROOT_GENERATE_DICTIONARY dictionary)
     target_compile_definitions(${dictionary} PRIVATE
       ${definitions} $<TARGET_PROPERTY:${ARG_MODULE},COMPILE_DEFINITIONS>)
 
+    # remove all -I prefixes from list of include dirs
+    set(pureincdirs)
+    foreach(dir ${includedirs})
+      string(SUBSTRING ${dir} 2 -1 dir0)
+      set(pureincdirs ${pureincdirs} ${dir0})
+    endforeach()
```

Comment:
Why add `-I` above, then remove here? Isn't `${incdirs}` already what you need?

Reply:
Again - `${incdirs}` is not complete list. There are other entries